tests/core/test_taipy.py

```diff
@@ -544,40 +548,19 @@ def test_cancel_job(self):
             tp.cancel_job("job_id")
             mck.assert_called_once_with("job_id")
 
-    def test_block_config_when_core_is_running_in_development_mode(self):
-        input_cfg_1 = Config.configure_data_node(id="i1", storage_type="pickle", default_data=1, scope=Scope.SCENARIO)
-        output_cfg_1 = Config.configure_data_node(id="o1", storage_type="pickle", scope=Scope.SCENARIO)
-        task_cfg_1 = Config.configure_task("t1", print, input_cfg_1, output_cfg_1)
-        scenario_cfg_1 = Config.configure_scenario("s1", [task_cfg_1], [], Frequency.DAILY)
-
-        with mock.patch("sys.argv", ["prog"]):
-            core = Core()
-            core.run()
-
-        scenario_1 = tp.create_scenario(scenario_cfg_1)
-        tp.submit(scenario_1)
-
-        with pytest.raises(ConfigurationUpdateBlocked):
-            Config.configure_scenario("block_scenario", set([task_cfg_1]))
-        core.stop()
```

Comment:
Why are we removing this test?

Reply:
We don't need to submit a scenario to test that the config is blocked. So whatever it is standalone or development, the config is blocked.
That means this test `test_block_config_when_core_is_running_in_development_mode` and the following one `test_block_config_when_core_is_running_in_standalone_mode` are doing the same.
So I kept only one and renamed it.

Reply:
That's reasonable. Thank you